app/assets/javascripts/discourse/app/models/site.js

```diff
@@ -139,6 +139,21 @@ const Site = RestModel.extend({
       return newCategory;
     }
   },
+
+  allowPersonalMessages(user) {
```

Comment:
I am not sure if site is the best place for this, feel free to suggest somewhere else.